src/WebManager/Azure.ContainerRegistry/MediaTypeHelper.cs

```diff
@@ -0,0 +1,22 @@
+namespace Azure.ContainerRegistry
```

Comment:
Could this be the following instead?
```suggestion
namespace WebManager.Utility
```

Reply:
Fixed